src/pages/data/data.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Data = () => {
```

Comment:
components should be in the same case and name as their file which should be in `PascalCase` (i.e. `Data` should be in `Data.jsx`, which is inside of `src/pages/Data`)

this also applies to the rest of the pages that we've defined here (`importcsv` becomes `ImportCSV`, `profile` becomes `Profile`, etc.)